### PR TITLE
[LMS-353] Update harmonizer for LMSX model changes

### PR DESCRIPTION
**Pull request by bradbanister**

### CODE REVIEW

File: src/lms-harmonizer/edfi_lms_harmonizer/helpers/constants.py [hunk 1]
```diff
@@ -4,3 +4,15 @@
...
+class SOURCE_SYSTEM:
+    GOOGLE = "Google"
+    SCHOOLOGY = "Schoology"
+    CANVAS = "Canvas"
+
+
+class SOURCE_SYSTEM_NAMESPACE:
+    GOOGLE = "uri://google.com/classroom"
+    SCHOOLOGY = "uri://schoology.com"
+    CANVAS = "uri://instructure.com/canvas"
```
Comment (stephenfuqua) [inline src/lms-harmonizer/edfi_lms_harmonizer/helpers/constants.py:18]: We need to make sure that we document these namespace requirements for any API clients. Perhaps we can do that in LMS-298 "Define security requirements". I'll make a note in that ticket.

File: src/lms-harmonizer/tests_integration_sql/orchestrator.py [hunk 2]
```diff
@@ -132,8 +132,13 @@ def _load_lms_extension_scripts(config: ServerConfig):
 
 
 def _load_ordered_scripts(config: ServerConfig, script_path: str):
+    files_in_path: List[str] = [
+        f
+        for f in os.listdir(script_path)
+        if os.path.isfile(os.path.join(script_path, f))
+    ]
```
Comment (stephenfuqua) [inline src/lms-harmonizer/tests_integration_sql/orchestrator.py:139]: Good change, avoiding accidentally grabbing random sub-directories.

**Review by stephenfuqua — COMMENTED** [review]
Initial review: looks good. I need to develop and execute a test plan before approving and merging.

File: src/lms-harmonizer/tests_integration_sql/orchestrator.py (hunk 2 above)
Comment (bradbanister) [reply src/lms-harmonizer/tests_integration_sql/orchestrator.py:139]: Yeah, I got bit by that during testing. 😎

File: src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py [hunk 3]
```diff
@@ -20,7 +20,7 @@
 from tests_integration_sql.orchestrator import run_harmonizer
 
 
-SOURCE_SYSTEM = "Test_LMS"
+SOURCE_SYSTEM = "Canvas"
```
Comment (stephenfuqua) [inline src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py:23]: Now that there are separate blocks of code for each LMS, can you add explicit tests for each of three, and a fourth scenario where the source system is unknown (and thus won't be harmonized)?

**Review by stephenfuqua — CHANGES_REQUESTED** [review]

File: src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py (hunk 3 above)
Comment (bradbanister) [reply src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py]: Done

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**